ci/update-certificates.sh

```diff
@@ -83,13 +83,25 @@ for row in $(echo $mapping | jq -c '.[]'); do
 
   # store password in secrets.$key yml
   spruce json secrets-updated/secrets.yml \
-  | jq --arg password "$(cat /dev/urandom | LC_ALL=C tr -dc "a-zA-Z0-9" | head -c 32)" ".secrets.${key} = \$password" \
+  | jq --arg password "$(cat /dev/urandom | LC_ALL=C tr -dc "a-zA-Z0-9" | head -c 32)" ".${ENVIRONMENT}_${key} = \$password" \
     | spruce merge \
     > secrets-updated/tmp.yml
   mv secrets-updated/tmp.yml secrets-updated/secrets.yml
 
 done
 
+# generate new secrets passphrase each time we update secrets
```

Comment:
Not quite sure how to tackle this for `secret-rotation` pipeline yet. The rest will need a `secret-rotation-*` anyway to get their new secrets.